server/route.go

```diff
@@ -64,7 +64,6 @@ func reactRootHandler(rw http.ResponseWriter, req *http.Request) {
 }
```

Comment:
@danvixent 

1. We should change the `/ui/organisations/{orgID}/groups` too to become `/projects`
2. There's a deprecated route we should remove as well -- `/ui/organisations/{orgID}/security`
3. We should make the API route change backwards compatible by creating a shim API called `/groups` that still forwards requests to the `/projects`